tensorflow/lite/testing/generate_examples_lib.py

```diff
@@ -3758,6 +3758,25 @@ def make_slice_tests(options):
           "begin": [[0, 0], [1, 0]],
           "size": [[2, 3], [2, 2]],
       },
+      # 4-D with size -1
+      {
+          "dtype": [tf.float32, tf.int32, tf.int64],
```

Comment:
For this case let's just do dtype: tffloat32, and index_type: tf.int32.

Otherwise looks great!

Reply:
@jdduke, I have changed this